### PR TITLE
Add more regexp examples to the Filebeat docs

### PR DESCRIPTION
**Pull request by dedemorton**
This resolves issue #926

### CODE REVIEW

**Comment by urso** [comment]
LGTM, but I'd like to test the regexes ones more.

File: filebeat/docs/multiline.asciidoc [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+       at com.example.myproject.Author.getBookIds(Author.java:35)
+       ... 1 more
+-------------------------------------------------------------------------------------
+
+To consolidate these lines into a single event in Filebeat, use the following multiline configuration:
+
+["source","sh",subs="attributes,callouts"]
+-------------------------------------------------------------------------------------
+multiline:
+    pattern: '^[[:space:]]+(at|...)|^Caused by:'
```
Comment (urso) [inline filebeat/docs/multiline.asciidoc]: this regex should become `^[[:space:]]+(at|\.\.\.)|^Caused by:` The `.` operator means, match any character. Due to requiring indentation in general, the pattern can be simplified to `^[[:space:]]+|^Caused by:` or `^[ \t]+|^Caused by:` .

File: filebeat/docs/multiline.asciidoc [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+(/dev/disk1)]], net usable_space [34.5gb], net total_space [118.9gb], types [hfs]
+-------------------------------------------------------------------------------------
+
+To consolidate these lines into a single event in Filebeat, use the following multiline configuration:
+
+[source,json]
+[source,yaml]
+-------------------------------------------------------------------------------------
+multiline:
+  pattern: '^\[[[:digit:]]{4}-[[:digit:]]{2}-[[:digit:]]{2}'
```
Comment (urso) [inline filebeat/docs/multiline.asciidoc]: alternative regex `^\[[0-9]{4}-[0-9]{2}-[0-9]{2}` or `^\[[0-9]{4}(-[0-9]{2}){2}`

File: filebeat/docs/multiline.asciidoc [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+
+* the `pattern` option, which specifies a regular expression. Depending on how you configure other multiline options, 
+lines that match the specified regular expression are considered either continuations of a previous line or the start of a new multiline event. You can set the `negate` option to negate the pattern.
+
+* the `match` option, which specifies how Filebeat combines matching lines into an event. You can specify `before` or `after`.
+
+See the full documentation for <<multiline>> to learn more about these options. Also read <<yaml-tips>> and 
+<<regexp-support>> to avoid common mistakes.
+
+==== Testing Your Regexp Pattern for Multiline
```
Comment (ruflin) [inline filebeat/docs/multiline.asciidoc]: We should definitively link to this in https://www.elastic.co/guide/en/beats/filebeat/master/regexp-support.html

Comment (dedemorton) [inline filebeat/docs/multiline.asciidoc]: It's there! <<regexp-support>> resolves to regexp-support.html. :-)

Comment (ruflin) [inline filebeat/docs/multiline.asciidoc]: @dedemorton Not sure if we mean the same. I don't mean to link from here to regexp-support, but link from regexp-support to here.

Comment (dedemorton) [inline filebeat/docs/multiline.asciidoc]: Ah, I misread. I will make sure to include the link there.

**Comment by dedemorton** [comment]
@urso I've updated the examples. I used the simpler pattern that recommended in your first comment. For the other example, I used ^\[[0-9]{4}-[0-9]{2}-[0-9]{2} because I felt it was easier to understand at a glance.

**Pull request merged**